lib/web_ui/lib/src/engine/text_editing/input_type.dart

```diff
@@ -14,7 +14,7 @@ import '../dom.dart';
 abstract class EngineInputType {
   const EngineInputType();
 
-  static EngineInputType fromName(String name, {bool isDecimal = false}) {
+  static EngineInputType fromName(String name, {bool isDecimal = false, bool isMultiline= false}) {
```

Comment:
nit: change `isMultiline= false` to `isMultiline = false`

Reply:
Sorry about this. Now I have fixed it. Thank you!

Reply:
@htoor3, Happy New Year! [The framework PR](https://github.com/flutter/flutter/pull/140356) has been merged. Is it time to merge this PR?